src/graphql/util.js

```diff
@@ -316,3 +316,18 @@ export function filterArticleRepliesByStatus(articleReplies, status) {
     );
   });
 }
+
+export function filterArticleCategoriesByStatus(articleCategories, status) {
+  if (!status) return articleCategories;
+
+  // If a articleCategory does not have status, it is considered "NORMAL".
```

Comment:
Actually this is considered a workaround, because "delete article reply" is introduced later than creating article reply, and I was too lazy doing migration.

Since we are rolling out article-category creation & deletion at the same time, we should not need this as long as article-category creation always includes a valid status field?